test/index.ts

```diff
@@ -610,16 +610,14 @@ describe('.getToken()', () => {
 });
 
 function createGetTokenMock(code = 200, body?: {}) {
-  return nock(GOOGLE_TOKEN_URLS[0])
+  return nock(GOOGLE_TOKEN_URLS[0], {
+    reqheaders: {'Content-Type': 'application/json'},
```

Comment:
we had `reqheaders` in the wrong place, and it was not actually being asserted against.

`application/x-www-form-urlencoded` was already being replaced by gaxios with `application/json`.

Reply:
I see that the docs do call out that headers are specified in this outer function, but doesn't it have to have some way to define header assertions on a per-route basis?  Like it makes sense to be that these could be defined at the top level for the host, but I would expect `.get` and `.post` to accept this parameter as well.  I am le nervous here.